app/helpers/cases_helper.rb

```diff
@@ -272,10 +272,25 @@ def action_link_for_destroy_case_link(kase, linked_case)
     end
   end
 
+  # def request_details_html(kase)
```

Comment:
I think we should delete this comment if we aren't using this implementation, could be confusing to someone looking back at the code 👍

Reply:
Commented code now removed.